app/src/main/res/layout/profile_chooser_fragment.xml

```diff
@@ -89,14 +87,15 @@
     </androidx.core.widget.NestedScrollView>
 
     <View
+      android:id="@+id/view"
```

Comment:
If you decide to make the above change then remove this ID too.

Reply:
Noted